onnxruntime/contrib_ops/cuda/bert/tensorrt_fused_multihead_attention/mha_runner.cu

```diff
@@ -29,6 +29,7 @@ union __half2_uint32_t_union {
 
 void set_alpha_fp16(uint32_t& alpha, float norm) {
   __half2_uint32_t_union temp;
+  memset(&temp, 0, sizeof(__half2_uint32_t_union));
```

Comment:
It fixed a bug. Part of the temp was not initialized.

Reply:
It is union so it is fine to only initialize the first one (the other one shared same memory).

Reply:
The union has two members:
```
 half2 fp162;
  uint32_t u32;
```
The first one is 2 bytes. The second one is 4 bytes. You only initialized the first 2 bytes.